petisco/extra/sqlalchemy/sql/sql_session_scope_provider.py

```diff
@@ -17,19 +17,14 @@ def session_scope() -> Iterator[Session]:
         try:
             yield session
             session.commit()
-        except OperationalError as e:
-            logger.error(e)
-            session.rollback()
-            raise e
         except WaitingForEarlyReturn as e:
             session.rollback()
             raise e
-        except Exception as e:
+        except (OperationalError, Exception) as e:
             logger.error(e)
             session.rollback()
             raise e
         finally:
             session.close()
-        # session.delete()
```

Comment:
@franciscorode @LdeBerried @fgsalomon 

I've deleted the commented code.

Info: https://docs.sqlalchemy.org/en/20/orm/session_basics.html#deleting

Should we keep it or no?

Reply:
Yo no sé en que momento se metió ese `delete`. De hecho creo que no hace nada porque habría que pasarle un objeto como [parámetro](https://docs.sqlalchemy.org/en/20/orm/session_api.html#sqlalchemy.orm.Session.delete).
Se comentó en el cambio a v2.
Yo lo quitaría sí.